src/engine.rs

```diff
@@ -276,6 +276,18 @@ impl Engine {
         Ok(())
     }
 
+    /// Sanitizes the stored data.
+    ///
+    /// This goes over the stored data and deletes what looks broken. It
+    /// should be called before retrying a failed restartable run.
+    pub fn sanitize(&self) -> Result<(), Fatal> {
+        self.store.sanitize()?;
+        if let Some(collector) = self.collector.as_ref() {
+            collector.sanitize()?;
+        }
+        Ok(())
+    }
+
```

Comment:
This still doesn't do anything, right? The .sanitize() method that is called doesn't do anything

Reply:
The RRDP collector’s sanitize method checks for the consistency of all its archives.